content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/index.mdx

```diff
@@ -32,18 +32,20 @@ In order to successfully complete this guide, you need:
 
 </SwitchTech>
 
+> # See [System requirements](/reference/system-requirements) for exact version requirements.
+
```

Comment:
This renders a bit weird for some reason: https://deploy-preview-2140--prisma2-docs.netlify.app/docs/getting-started/setup-prisma/start-from-scratch/relational-databases-typescript-postgres
![image](https://user-images.githubusercontent.com/183673/133077938-a9a61785-8c22-48ad-b5d6-fd95e2b1d2d5.png)
(circled grey thing and the added distance vs. current version)

Reply:
Ahhh, I think this sentence is duplicated! I see if below under the last tech switcher for sqlserver again.

Reply:
That is odd, will check the file